doc/python/v6-changes.md

```diff
@@ -0,0 +1,137 @@
+---
+jupyter:
+  jupytext:
+    notebook_metadata_filter: all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.4
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+  language_info:
+    codemirror_mode:
+      name: ipython
+      version: 3
+    file_extension: .py
+    mimetype: text/x-python
+    name: python
+    nbconvert_exporter: python
+    pygments_lexer: ipython3
+    version: 3.11.10
+  plotly:
+    description: Guide to changes in version 6 of Plotly.py and how to migrate from
+      version 5
+    display_as: file_settings
+    language: python
+    layout: base
+    name: Changes in Version 6
+    order: 8
+    page_type: example_index
+    permalink: python/v6-migration/
+    thumbnail: thumbnail/v4-migration.png
+---
+
+This page outlines the changes in Plotly.py version 6 and cases where you may need to update your charts or tools that you use for working with Plotly.py.
+
+<!-- #region -->
+## Jupyter Notebook Support
+
+Versions of Jupyter Notebook earlier than version 7 are no longer supported. To upgrade to the latest Jupyter Notebook:
+
+```
+pip install notebook --upgrade
+```
+
+## Change to anywidget for go.FigureWidget
```

Comment:
Is it worth mentioning here that `ipywidgets` is no longer a dependency? (maybe a question for @marthacryan ?)

Reply:
it's a dependency of `anywidget` so I'd say not worth mentioning.